x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_setup_trust/remote_cluster_setup_trust.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useContext } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiSpacer,
+  EuiCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiButton,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { AppContext } from '../../../app_context';
+import { ConfirmTrustSetupModal } from './confirm_modal';
+
+const CARD_MAX_WIDTH = 400;
+const i18nTexts = {
+  apiKeyTitle: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.title',
+    { defaultMessage: 'API keys' }
+  ),
+  apiKeyBadge: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.badge',
+    { defaultMessage: 'BETA' }
+  ),
+  apiKeyDescription: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithApiKeys.description',
+    {
+      defaultMessage:
+        'Set up your authentication mechanism with API Keys to connect to your remote cluster.',
+    }
+  ),
+  certTitle: i18n.translate('xpack.remoteClusters.clusterWizard.trustStep.setupWithCert.title', {
+    defaultMessage: 'Certificates',
+  }),
+  certDescription: i18n.translate(
+    'xpack.remoteClusters.clusterWizard.trustStep.setupWithCert.description',
+    {
+      defaultMessage:
+        'This is our easier to setup method, but less secure than the API key based approach.',
+    }
+  ),
+};
+
+const docLinks = {
+  apiKey:
+    'https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html',
+  apiKeyCloud:
+    'https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html',
+  cert: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html',
+  certCloud:
+    'https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html',
+};
+
+interface Props {
+  onBack: () => void;
+  onSubmit: () => void;
+  isSaving: boolean;
+}
+
+export const RemoteClusterSetupTrust = ({ onBack, onSubmit, isSaving }: Props) => {
+  const [isModalVisible, setIsModalVisible] = useState<boolean>(false);
+  const { canUseAPIKeyTrustModel, isCloudEnabled } = useContext(AppContext);
+
+  return (
+    <div>
+      <EuiText size="m" textAlign="center">
+        <p>
+          <FormattedMessage
+            id="xpack.remoteClusters.clusterWizard.trustStep.title"
+            defaultMessage="To establish trust between this deployment and your newly created remote cluster, {br} choose one of the two options below."
```

Comment:
```suggestion
            defaultMessage="To establish trust between this deployment and your newly created remote cluster, {br} choose one of the following options."
```
Replace "below" with a more accessible term. Check [How to create accessible web content](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit).